mkwikitable

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+
+from argparse import ArgumentParser
+from socket import AF_INET, AF_INET6, inet_pton
+import jinja2
+import json
+from filereader import get_communities_data
+from apireader import get_api_dict, get_api_data
+
+templates = {"ip-netze.j2": """====IPv6====
+{| class="wikitable sortable"
+|- bgcolor="#efefef"
+! Netzwerk
+! Community
+! AS
+! Kontakt (tech-c)
+{% for network in ipv6networks %}
+|-
+| {{ network.network }}
+{%- if network.community_url != "" %}
+| [{{ network.community_url }} {{ network.community_name }}]
+{%- else %}
+| {{ network.community_name }}
+{%- endif %}
+| {{ network.asn }}
+| {% for mail in network.techc -%}
+{{- "[mailto:%s Mail]"|format(mail)|replace('@','--at--') -}}
+{%- if not loop.last %}, {% endif -%}
+{%- endfor -%}
+{%- endfor %}
+|- bgcolor="#efefef"
+! Netzwerk
+! Community
+! AS
+! Kontakt (tech-c)
+|}
+
+====IPv4====
+{| class="wikitable sortable"
+|- bgcolor="#efefef"
+! Netzwerk
+! Community
+! AS
+! Kontakt (tech-c)
+{% for network in ipv4networks %}
+|-
+| {{ network.network }}
+{%- if network.community_url != "" %}
+| [{{ network.community_url }} {{ network.community_name }}]
+{%- else %}
+| {{ network.community_name }}
+{%- endif %}
+| {{ network.asn }}
+| {% for mail in network.techc -%}
+{{- "[mailto:%s Mail]"|format(mail)|replace('@','--at--') -}}
+{%- if not loop.last %}, {% endif -%}
+{%- endfor -%}
+{%- endfor %}
+|- bgcolor="#efefef"
+! Netzwerk
+! Community
+! AS
+! Kontakt (tech-c)
+|}"""}
+
+# templates = {"ip-netze.j2":"""
+# {% for network in ipv4networks %}
+# {{- network.community_url -}}
+# {% endfor %}
+# """}
+
+
+def get_networklist(meta_src, api_src, family):
+    """
+    Returns a sorted list of all freifunk networks (family:ipv4/ipv6).
+    Each entry is a dict containing the network and
+    additional information about the network (asn, tech-c, community_name and community_url).
+    """
+    all_networks = []
+
+    api_dict = get_api_dict(api_src)
+    api_data = get_api_data(api_dict)
```

Comment:
You are currently fetching the metadata twice, for ipv4 and ipv6. This should be reduced to just once.